roles/openshift_loadbalancer/tasks/main.yml

```diff
@@ -68,5 +68,9 @@
     daemon_reload: yes
   register: start_result
 
+- name: Setup tuned
+  include: ../../tuned/tasks/main.yml
```

Comment:
If we're going to make tuned it's own role, we shouldn't include tasks from that role inside of other roles.  The role should be called directly from a playbook in the proper place.

Reply:
Thanks for feedback.  What would be the proper place?